docs_src/index.md

```diff
@@ -173,16 +174,22 @@ SDKs are language specific; meaning an SDK is written to create services in a pa
 ## How EdgeX Works
 
 ### Sensor Data Collection
-EdgeX’s primary job is to collect data from sensors and devices and make that data available to north side applications and systems.  Data is collected from a sensor by a device service that speaks the protocol of that device.  Example: a Modbus device service would communicate in Modbus to get a pressure reading from a Modbus pump.  The device service translates the sensor data into an EdgeX event object and sends the event object to the core data service via REST communications (step 1).
+EdgeX’s primary job is to collect data from sensors and devices and make that data available to north side applications and systems.  Data is collected from a sensor by a device service that speaks the protocol of that device.  Example: a Modbus device service would communicate in Modbus to get a pressure reading from a Modbus pump.  The device service translates the sensor data into an EdgeX event object.  The device service can then either:
+
+1. put the event object on a message bus (which may be implemented via Redis Streams or MQTT).  Subscribers to the event message on the message bus can be application services or core data or both (see step 1.1 below).
+
+    ![image](./general/EdgeX_step1-1.png)
 
-![image](./general/EdgeX_step1.png)
+2. send the event object to the core data service via REST communications (see step 1.2).
 
-Core data persists the sensor data in the local edge database.  Redis is used as the database by default (other databases can be used alternatively).  In fact, persistence is not required and can be turned off.  Data is persisted in EdgeX at the edge for two basics reasons:
+    ![image](./general/EdgeX_step1-2.png)
+
+When core data receives the event (either via message bus or REST), it persists the sensor data in the local edge database.  Redis is used as the database by default (other databases can be used alternatively).  Persistence is not required and can be turned off.  Data is persisted in EdgeX at the edge for two basics reasons:
```

Comment:
I thing we should remove `(other databases can be used alternatively)` as it sounds like we have other DB options that can be selected.